### PR TITLE
CLF ERT refresh

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Updates the CLF ERT with new loadouts.

This isn't particularly trying to make CLF more powerful (although it probably does do that), but mainly give them more variety.
The CLF are impoverished angry colonists, so presumably have to scrounge up whatever weaponry they can get, or buy stuff off the UPP/SOM where they can.

To this end, added a whole bunch of different loadouts to spice them up a bit. Oh also they have lights on their suits now. woo.

Additionally, tweaked the balance of the Skorpion, which was utter dog shit. It's still bad, but hopefully isn't entirely useless now.

Also a few minor sprite fixes etc.

## Why It's Good For The Game
Variety is good.

## Changelog
:cl:
add: The CLF now carry a much larger variety of equipment
balance: Improved the performance of the Skorpion SMG
/:cl:

### CODE REVIEW

**Comment by MereWalker** [comment]
The scorpion is crazy good vs marines the last time I used it. Which was a long time ago. Could these be used in combat as well? Like a factions system.

**Comment by Lumipharon** [comment]
> The scorpion is crazy good vs marines the last time I used it. Which was a long time ago. Could these be used in combat as well? Like a factions system.

Dunno if it was changed in the past, but right now it's hot garbage.

File: code/modules/clothing/under/marine_uniform.dm [hunk 1]
```diff
@@ -319,6 +319,8 @@
 	icon_state = "colonist"
 	has_sensor = 2
 
+/obj/item/clothing/under/colonist/webbing
+	starting_attachments = list(/obj/item/armor_module/storage/uniform/black_vest)
```
Comment (BraveMole) [inline code/modules/clothing/under/marine_uniform.dm:323]: Spacing

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/modules/clothing/under/marine_uniform.dm (hunk 1 above)
Comment (Lumipharon) [reply code/modules/clothing/under/marine_uniform.dm:323]: done

**Pull request merged**